### PR TITLE
Update pre-commit Hooks

### PR DESCRIPTION
**Pull request by mcdonnnj**
# 

## 🗣 Description

This PR:
1. Removes the [seed-isort-config](https://github.com/asottile/seed-isort-config) hook as it has been deprecated.
1. Has updates from running `pre-commit autoupdate`
1. Has a small change to the contents of `CONTRIBUTING.md`

## 💭 Motivation and Context

Keeping hooks up-to-date is good. Removing hooks that are no longer needed/functional is also good. Clearer documentation is _even more_ good.

## 🧪 Testing

Automated tests pass. I confirmed in [cisagov/skeleton-python-library](https://github.com/cisagov/skeleton-python-library) that the `isort` changes do not change expected functionality.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ skeleton projects derived from this one.
 
 ## Contributing ##
 
-We welcome contributions!  Please see [here](CONTRIBUTING.md) for
+We welcome contributions!  Please see [`CONTRIBUTING.md`](CONTRIBUTING.md) for
```
Comment (jsf9k) [inline README.md:24]: :+1:

**Review by jsf9k — APPROVED** [review]
Thanks for keeping thangs updated.  If only you could work the same magic on my wardrobe, slang, and hairstyle!

**Review by dav3r — APPROVED** [review]
Fantastico! 💎

**Review by hillaryj — COMMENTED** [review]
Looks great! 🐙

**Pull request merged**